project/AppDependencies.scala

```diff
@@ -6,13 +6,13 @@ object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc"            %% "bootstrap-backend-play-27" % "2.14.0",
+    "uk.gov.hmrc"            %% "bootstrap-backend-play-28" % "4.1.0",
     "org.scala-lang.modules" %% "scala-xml"                 % "1.3.0"
   )
 
   val test = Seq(
     "org.scalatest"          %% "scalatest"          % "3.1.1"             % s"$Test,$IntegrationTest",
```

Comment:
Scalatest can be removed as it is included in scalatestplus-play: https://mvnrepository.com/artifact/org.scalatestplus.play/scalatestplus-play_2.12/5.1.0

Reply:
Cool - Will update.

Reply:
Have updated.

I also realised that I could remove `play.http.filters` in application.conf too, as they're defined by backend.conf now. That showed up as a warning when running the app.